schematic_api/api/openapi/api.yaml

```diff
@@ -9,12 +9,20 @@ info:
 servers:
   - url: /v1
 
+components:
```

Comment:
In the future, it's important to think about your PRs.  Are these actually two PRs? Here you are doing two distinct things.

1. Adding API key support
2. Renaming the manifest path

Having smaller, more, but higher quality PRs with tightly scoped features allow for faster and more succinct reviews.

Reply:
Right. Thank you for pointing it out. I will be careful next time.